_includes/domain-availability/domain-availability-search-input.html

```diff
@@ -0,0 +1,20 @@
+{% comment %}
+Domain availability search. Informs user if an inputted domain name
+is available as a .gov domain.
+{% endcomment %}
+
+<form class="usa-search usa-search--domain search-whois" role="search">
+    <label class="usa-sr-only" for="domain-input">Search</label>
+    <div class="usa-search--domain__form-group">
+        <input
+            class="usa-input usagov-search-autocomplete" 
+            id="domain-input"
+            type="search" 
+            name="query" 
+            autocomplete="off"
+            aria-label="Search domain availability"
+            title="Search domain availability"
+        >
+        <span class="usa-search__submit-text">.gov</span>
+    </div>
```

Comment:
The search box is a bit confusing. It looks like the .gov is being supplied by the search box, but actually it's required. Maybe this is a question for design, but should we auto-append the required .gov at the end?

Reply:
Update: Talked with design and the search supports inputs with or without appending .gov as the TLD
![image](https://github.com/user-attachments/assets/bd1918c3-0b8b-417b-9fc8-3d5c694db0c4)
![image](https://github.com/user-attachments/assets/89d40728-5150-4673-b67c-fe9be0728905)